generate.py

```diff
@@ -25,7 +26,7 @@ def header_files(include_paths):
     for dir in include_paths:
         if os.path.exists(dir):
             break
-    return {fn: os.path.join(dir, fn) for fn in os.listdir(dir)}
+    return {fn: os.path.join(dir, fn) for fn in os.listdir(dir) if fnmatch.fnmatch(fn, '*.h')}
```

Comment:
I'd prefer to just use `.endswith('.h')` for this, so we don't need the extra import. Also, is this line still shorter than 80 characters?